net/release-notes/release-notes-2023/aspose-psd-for-net-23-8-release-notes/_index.md

```diff
@@ -0,0 +1,219 @@
+---
+title: Aspose.PSD for .NET 23.8 - Release Notes
+type: docs
+weight: 50
+url: /net/aspose-psd-for-net-23-8-release-notes/
+---
+
+{{% alert color="primary" %}}
+
+This page contains release notes for [Aspose.PSD for .NET 23.8](https://www.nuget.org/packages/Aspose.PSD/)
+
+{{% /alert %}}
+
+| **Key**     | **Summary**                                                                                                  | **Category** |
+|:------------|:-------------------------------------------------------------------------------------------------------------|:--------|
+| PSDNET-1566 | Add new type of warp (Flag) | Feature |
+| PSDNET-1621 | Add new types of warp: arc up, arc down, sphere | Feature |
+| PSDNET-1682 | Implement new method PsdImage.AddPosterizeAdjustmentLayer for adding new Posterize layer | Feature |
+| PSDNET-913  | PSD information lost on just opening and saving | Bug     |
+| PSDNET-1352 | Image loading failed | Bug     |
+| PSDNET-1553 | Image loading failed: Unable to cast object of type UnknownStructure to type DescriptorStructure | Bug     |
+| PSDNET-1622 | Examine increased memory consumption by test 189 on Jenkins build | Bug     |
```

Comment:
Could you please remove this item. This internal issue.

Reply:
done